Plugins~/Src/MeshSyncClientBlender/python/3.3.0/unity_mesh_sync.py

```diff
@@ -380,6 +382,10 @@ def meshsync_post_export():
     for f in mesh_sync_on_post_export:
         f()
 
+def meshsync_server_requested_callback():
+    for f in mesh_sync_server_requested_callback:
+        f()
```

Comment:
might be easier to move this code to another file, perhaps unity_meshsync_common or a new module, so we don't have to edit multiple files for one code change

Reply:
Done now